# HG changeset patch
# User Sean Farley <sean.michael.farley@gmail.com>
# Date 1397847476 18000
#      Fri Apr 18 13:57:56 2014 -0500
# Node ID ba9baa3e45905e06f670f7c19198c0b7190c9706
# Parent  7a1883c3f9ed3e8c9b9f97de7ecb5ed7e37c058a
make: use variable for stdcxxlib

diff --git a/admin/Makefile.build b/admin/Makefile.build
--- a/admin/Makefile.build
+++ b/admin/Makefile.build
@@ -372,6 +372,8 @@ endif
 -lm:
 
 # Required by ASDM libs:
 -lstdc++:
 
+-lc++:
+
 ###########################################################################
diff --git a/admin/Makefile.def b/admin/Makefile.def
--- a/admin/Makefile.def
+++ b/admin/Makefile.def
@@ -531,11 +531,11 @@ endif
 # various systems
 ifeq ($(GAG_USE_32BITS),yes)
   # Compilation in 32 bits mode
   GAG_32BITS_SWITCH = "-m32"
 endif
-STDCpp_LIB += $(shell g++ $(GAG_32BITS_SWITCH) -print-file-name=libstdc++.$(LIB_CPP_EXT))
+STDCpp_LIB += @STDCXXLIB@
 
 ###########################################################################
 # Kernel libraries
 
 LEGACY_LIBS = -lslatec -llapack -lblas
